### PR TITLE
Cleanup vm state machine

### PR DESCRIPTION
**Pull request by gao-feng**

### CODE REVIEW

File: hypervisor/devicemap.go [hunk 1]
```diff
@@ -361,27 +360,6 @@ func (ctx *VmContext) netdevInserted(info *NetDevInsertedEvent) {
...
-				glog.V(1).Info("need remove image dm file", image.info.Filename)
-				ctx.progress.deleting.blockdevs[name] = true
-				go UmountDMDevice(image.info.Filename, name, ctx.Hub)
-			}
-		}
-	}
-
-	return c.Success
-}
-
```
Comment (laijs) [inline hypervisor/devicemap.go]: why onContainerRemoved() is also removed?

Comment (laijs) [inline hypervisor/devicemap.go]: OK, it is unused currently. but it is possible needed in future.

Comment (gao-feng) [inline hypervisor/devicemap.go]: then add them back if necessary.

**Comment by gnawux** [comment]
do all the volume cleanup code are removed? I wonder if this will 

- leave the dm devices in `/dev` after the pod stopped
- leave overlay/aufs mount dirs after the pod stopped

**Comment by gao-feng** [comment]
yes, hyperd will do the cleanup job. https://github.com/hyperhq/hyperd/pull/303

**Pull request merged**